metrics/health.go

```diff
@@ -44,7 +47,15 @@ type (
 )
 
 var (
-	healthStatus = sync.Map{}
+	healthStatus     = sync.Map{}
+	PromHealthStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "component_health_status",
```

Comment:
Let's add `pelican_` as a prefix; otherwise, this is a pretty generic metric name.